test/config.js

```diff
@@ -15,15 +15,11 @@ config.views.bundle.packages.push({
   manifest: path.join(__dirname, 'package.json')
 });
 
-const WorkerPlugin = require('worker-plugin');
-
+// FIXME: can we remove this entirely now?
```

Comment:
I believe so, yes.  If webpack can build the browser bundle without this, we're good.

Reply:
Removed.